docs/index.rst

```diff
@@ -24,6 +24,48 @@ database using the `PostgreSQL Wire Protocol`_.
 databases in Java.
 
 
+.. _about:
+
+*****
+About
+*****
+
+Overview
+========
+
+For general purpose use, we recommend to use the official `PostgreSQL JDBC
+Driver`_.
+
+*This* JDBC driver is needed in certain scenarios like the one outlined at
+`Apache Kafka, Apache Flink, and CrateDB`_. The background is that, when using
+the ``postgresql://`` JDBC driver prefix, the `Apache Flink JDBC Connector`_
+will implicitly select the corresponding dialect implementation for PostgreSQL.
+
+In turn, this will employ a few behaviours that strictly expect a PostgreSQL
+server on the other end, so that some operations will croak on databases
+offering wire-compatibility with PostgreSQL, but do not provide the
+corresponding features like the `hstore`_ or `jsonb`_ extensions.
```

Comment:
"certain features" instead of "corresponding features" perhaps?

Reply:
Addressed with 1684ea59, thanks!